### PR TITLE
Native YAML - cloud/softlayer/sl_vm.py

### PR DESCRIPTION
**Pull request by Fale**
##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
cloud/softlayer/*

##### ANSIBLE VERSION
```
2.2
```

##### SUMMARY
Make YAML code use the native style

@gundalow

### CODE REVIEW

**Comment by Fale** [comment]
Migrated from https://github.com/ansible/ansible-modules-extras/pull/3629

File: lib/ansible/modules/cloud/softlayer/sl_vm.py [hunk 1]
```diff
@@ -187,17 +185,49 @@
...
+        os_code: UBUNTU_LATEST
+        ssh_keys: []
+        wait: True
 
 - name: Cancel instances
   hosts: localhost
   gather_facts: False
   tasks:
   - name: Cancel by tag
-    local_action:
```
Comment (gundalow) [inline lib/ansible/modules/cloud/softlayer/sl_vm.py]: Why has this been removed?

Comment (Fale) [reply lib/ansible/modules/cloud/softlayer/sl_vm.py]: Because the hosts is localhost

**Comment by gundalow** [comment]
Thank you for being very proactive in your efforts to migrate your PR, however it appears that there was a window of time when ansible/ansible:devel was broken and it appears, unfortunately, that your PR was migrated during that window.

Due to this you will need to follow a hopefully very simple process to fix this. Please see https://groups.google.com/forum/#!topic/ansible-devel/0eRSpiGkios

As a thank you for being pioneers, we will actively review your PRs as a priority. Once you've migrated your PR please add it to the Core Agenda https://github.com/ansible/community/issues/147

**Pull request closed without merging**